### PR TITLE
add some tests

### PR DESCRIPTION
**Pull request by nlehuby**

### CODE REVIEW

File: tests/test.rs [hunk 1]
```diff
@@ -0,0 +1,44 @@
+extern crate extract_validity_period;
+
+fn test_wrapper(path_to_txt:&str, expected_result:(u32,u32)) {
+    let init_path = std::env::current_dir().unwrap();
+    let test_path = std::path::Path::new(path_to_txt);
+    assert!(std::env::set_current_dir(&test_path).is_ok());
```
Comment (TeXitoi) [inline tests/test.rs]: You can just `unwrap()`, the error message will be better.

File: tests/test.rs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+    let result = extract_validity_period::get_period_from_txt_files();
+    assert_eq!(result, expected_result);
+
+    assert!(std::env::set_current_dir(&init_path).is_ok());
```
Comment (TeXitoi) [inline tests/test.rs]: same

File: tests/test.rs [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    test_wrapper("tests/fixtures/empty_calendar", (20160504, 20160930))
+}
+
+#[test]
+fn test_empty_calendar_dates() {
+    test_wrapper("tests/fixtures/empty_calendar_dates", (20161107, 20170205))
+}
+
+#[test]
+#[ignore]
```
Comment (TeXitoi) [inline tests/test.rs]: Not passing tests?

Comment (nlehuby) [inline tests/test.rs]: yes, theses tests are in prevision to the correction of issue #2

**Comment by nlehuby** [comment]
I forgot to commit the fixtures calendar* files (due to a bad .gitignore) ...

**Comment by nlehuby** [comment]
It should work better with the fixtures ;)

**Comment by nlehuby** [comment]
can we merge this one ?

**Pull request merged**